kosogor-terraform/src/main/kotlin/tanvd/kosogor/terraform/tasks/validate/ValidateModulesTask.kt

```diff
@@ -45,32 +44,27 @@ open class ValidateModulesTask : DefaultTask() {
                         init(workingDir)
                         validate(workingDir)
                     } finally {
-                        File(file.parentFile, "terraform").deleteRecursively()
+                        File(file.parentFile, ".terraform").deleteRecursively()
                         providerFile?.delete()
                     }
                     println("Validated module ${packageInfo.group}:${packageInfo.name}:${packageInfo.version}")
                 }
     }
 
     private fun init(workingDir: File) {
-        val curInitDir = File(workingDir, "terraform")
-        if (terraformDsl.validater.cacheInitPlugins && GlobalFile.tfInitDir.exists()) {
```

Comment:
So, you've removed support of cached plugins? It will not make terraform re-download plugins each time?

Reply:
Fixed.